kolibri/core/assets/src/views/CoreMenu/index.vue

```diff
@@ -127,7 +127,7 @@
 
   @import '~kolibri-design-system/lib/styles/definitions';
   .ui-menu-header {
-    padding: 1rem 1rem 1rem 50px;
+    padding: 1rem 1rem 1rem 1.2rem;
```

Comment:
I feel like there might be a reason for this fixed pixel padding here, or does 1.2rem give the desired padding at all screen resolutions?

Reply:
I also suspect there might be a reason, but keeping it at `50px` now causes some very wonky alignment issues in the learner view. I've tests it in several places with different screen sizes and so far I don't see any issues with this change, but I definitely do not know the ins and outs of manual QA to know if there are _no_ problems. Also open to suggestions about how else I might set up the nested component here in a way that would keep me from making an update in `index.vue` - maybe @MisRob would have an idea...

Reply:
I'll have a look

Reply:
I assume that there indeed was a specific reason to have `50px` and that it was to make header text aligned with options. See the screenshot from `develop`:

![Screenshot from 2021-07-22 10-27-05](https://user-images.githubusercontent.com/13509191/126610516-41eeadd1-e2c7-44b1-ba8c-56b608e55d96.png)

According to the new designs for user syncing

![Screenshot from 2021-07-22 10-35-43](https://user-images.githubusercontent.com/13509191/126611286-dfb2a772-2838-463f-8ba5-3ceaa0b4cb67.png)

this is to be deprecated, at least in some cases, so this change looks fine to me.

However, I am not sure if we're supposed to keep the previous layout when no sync information is visible in the menu, like for the admin. If yes, this change will cause the following regression:

![Screenshot from 2021-07-22 10-33-43](https://user-images.githubusercontent.com/13509191/126611034-774b64ff-c984-49aa-8d52-682c4f1bcaf4.png)

I feel fine about using the new layout everywhere for the sake of consistency. Maybe let's see what designers think.

Reply:
In any case, it's not a blocker. I checked all places where we use `CoreMenu` and haven't noticed any other issues.